specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -8861,6 +9137,322 @@
           "description": "The latest restorable timestamp for a resource."
         }
       }
+    },
+    "ServiceResource": {
+      "description": "Service cluster response resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The service name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a Data Transfer cluster.",
+          "$ref": "#/definitions/ServiceClusterProperty"
```

Comment:
Customer doesnt understand Cluster. It is an internal detail. This should be called ServicesProperties.

Reply:
Let's remove the term cluster everywhere, I see some example file names contain the term.